src/ipc/mem/Segment.cc

```diff
@@ -11,6 +11,8 @@
 #include "squid.h"
 #include "base/TextException.h"
 #include "compat/shm.h"
+#include "compat/socket.h"
```

Comment:
The API provided by this include is not used in this .cc file.
```suggestion
```

Reply:
removed